numpy/core/tests/test_umath.py

```diff
@@ -31,6 +31,13 @@
     uf for uf in UFUNCS_UNARY if 'f->f' in uf.types
 ]
 
+UFUNCS_BINARY = [
+    uf for uf in UFUNCS if uf.nin == 2
+]
+UFUNCS_BINARY_ACC = [
+    uf for uf in UFUNCS_BINARY if hasattr(uf, "accumulate") and uf.nout == 1
```

Comment:
Btw. the `hasattr` shouldn't do anything right now.  Checking here that `ufunc.signature is None` would make more sense to me.
But its mainly a nitpick and the rest is important for backporting.

Reply:
Thanks for clarification, in that case, the following skip can be avoided:

https://github.com/numpy/numpy/blob/1a0ea79eae7da0dad90051bd3c5d7a8097b427f2/numpy/core/tests/test_umath.py#L4356-L4357